Library/Formula/nodebrew.rb

```diff
@@ -22,6 +22,6 @@ def caveats; <<-EOS.undent
   end
 
   test do
-    assert shell_output("#{bin}/nodebrew ls-remote").include?("v0.10.0")
+    assert_match version.to_s, shell_output("#{bin}/nodebrew --version | egrep \"nodebrew \\d\"")
```

Comment:
Maybe the audit complained about `assert...include`?

```ruby
assert_match /v0.10.0/, shell_output("#{bin}/nodebrew ls-remote")
```
Should work on that though.